CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.8.9 - 2018-11-02
+This is a maintenance release.
+
+Maintenance:
+  * Add fpm to Gemfile that gets included in package builds.
+  * Hard-code `rubylibdir` variable in install script
```

Comment:
suggestion: maybe a note as to why we had to hardcode it in? (also missing a period)

Reply:
That info is in the commit message. I'm never totally sure what level of detail to include here vs. there..

Reply:
I can try to include a short summary here.